cmd/context/utils.go

```diff
@@ -176,3 +185,187 @@ func LoadStackWithContext(ctx context.Context, name, namespace string, stackPath
 	s.Namespace = okteto.Context().Namespace
 	return s, nil
 }
+
+// GetManifest Loads a manifest
+func GetManifest(ctx context.Context, srcFolder string, opts *ManifestOptions) (*model.Manifest, error) {
```

Comment:
Minor detail: if opts never can be null and we don't check if it is nil or not, I wouldn't make it a pointer, I would use `ManifestOptions`.